flow-typed/redux/digitalAssets.js

```diff
@@ -164,12 +164,18 @@ declare type DigitalAssetsSendFormFields = {|
   +assetAddress: AssetAddress,
 |}
 
+declare type SendTransactionParams = {|
+  gasPrice: ?string,
+  gasLimit: ?string,
+|}
```

Comment:
what is `SendTransactionParams`?
and why there are only two optional fields for gas settings?
obviously it must be renamed for something like `DefaultGasOptions`, `TransactionDefaultGas`, `GasSettings`, etc

Reply:
No, if there will be something to add we will rename it. Currently it is gasOptions only. Lets write clear and understandable code, that people able to read.

Reply:
renamed to GasSettings / gasSettings
please resolve asap